crates/bevy_reflect/src/type_info.rs

```diff
@@ -20,7 +20,7 @@ use std::any::{Any, TypeId};
 /// # Example
 ///
 /// ```
-/// # use std::any::Any;
+/// # use std::any::{Any, TypeId};
```

Comment:
Can this be reverted back to the following or is it still needed?
```suggestion
/// # use std::any::Any;
```